appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyEnvironment.java

```diff
@@ -41,42 +41,44 @@
  */
 public class VmApiProxyEnvironment implements ApiProxy.Environment {
 
+  public static final String USE_GLOBAL_TICKET_KEY="GAE_USE_GLOBAL_TICKET";
```

Comment:
can you run the formatter on your changes. This should fix issues like missing space before and after '=' and also space after '?', ':' as well as few line wraps.